### PR TITLE
Various card fixes

### PR DESCRIPTION
**Pull request by gpfaia**
Added keep card does proper super
Added NovaCard to own folder type (fixed its references)
Added back drop from high altitude
Fixed energize card name

### CODE REVIEW

File: backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from game.cards.discard_card import DiscardCard
+from game.values.locations import Locations
+import game.turn_actions.player_movement as player_movement
+
+
+class DropFromHighAltitude(DiscardCard):
+    def __init__(self):
+        super().__init__("Drop from High Altitude", 5,
+                         "+ 2[Star] and take control of Tokyo if you don't already control it")
```
Comment (cvennel) [inline backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py]: Drop from high altitude has a footnote: "(player choose to take control either City & Bay if both are occupied) "

This also means you shouldn't be moving everyone out of tokyo

Comment (gpfaia) [reply backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py]: Right now we only have 1 Tokyo, the action i see is a force field would have to happen

**Comment by gpfaia** [comment]
closes #224, renames energizes correctly

File: backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py (hunk 1 above)
Comment (cvennel) [reply backend/game/cards/discard_cards/victory_point_manipulation_cards/drop_from_high_altitude.py]: To be improved in future issue

**Pull request merged**